package-lock.json

```diff
@@ -6436,6 +6436,15 @@
         "path-exists": "^3.0.0"
       }
     },
+    "keyring_js": {
```

Comment:
we don't have keyring_js in package.json (as direct dependency) of explorer jes, i believe this should not be here

Reply:
this **commit hash above not required 1dbffa2**
is not required